app/src/main/java/com/it3048/mobiledeviceproject/MainActivity.kt

```diff
@@ -1,38 +1,29 @@
 package com.it3048.mobiledeviceproject
 
-import android.content.DialogInterface
 import android.os.Bundle
-import android.view.ContextThemeWrapper
-import android.view.LayoutInflater
+import android.view.*
 import android.widget.*
-import android.view.Menu
-import android.view.MenuItem
-import android.widget.EditText
-import android.widget.Spinner
-import android.widget.LinearLayout
-import android.widget.Toast
-import androidx.appcompat.app.AlertDialog
 import androidx.appcompat.app.AppCompatActivity
 import androidx.recyclerview.widget.RecyclerView
 import com.google.android.material.floatingactionbutton.FloatingActionButton
-import com.google.android.material.snackbar.Snackbar
-import kotlinx.android.synthetic.main.activity_main.*
-import java.util.*
+import com.it3048.mobiledeviceproject.dto.Meeting
 import kotlin.collections.ArrayList
+import kotlin.random.Random
+import kotlin.random.Random.Default.nextInt
 
 
 class MainActivity : AppCompatActivity() {
 
     private var addMeetingForm = AddMeetingForm()
-    lateinit var recyclerView: RecyclerView
-    lateinit var meetingList:ArrayList<MeetingDAO>
-    lateinit var meetingAdapter: MeetingAdapter
-    val inflater = LayoutInflater.from(this)
-    val v = inflater.inflate(R.layout.meeting_card, null)
-    val meetingTitle = v.findViewById<TextView>(R.id.meetingNameTxt)
-    val meetingDate = v.findViewById<TextView>(R.id.meetingDateTxt)
-    val meetingDescription = v.findViewById<TextView>(R.id.meetingDescriptionTxt)
-    val meetingLink = v.findViewById<TextView>(R.id.meetingUrlTxt)
+    private lateinit var recyclerView: RecyclerView
+    private lateinit var meetingList: ArrayList<Meeting>
+    private lateinit var meetingAdapter: MeetingAdapter
+    private val inflater = LayoutInflater.from(this)
+    private val v: View = inflater.inflate(R.layout.meeting_card, null)
+    private val meetingTitle: TextView = v.findViewById<TextView>(R.id.meetingNameTxt)
+    private val meetingDate: TextView = v.findViewById<TextView>(R.id.meetingDateTxt)
+    private val meetingDescription: TextView = v.findViewById<TextView>(R.id.meetingDescriptionTxt)
+    private val meetingLink: TextView = v.findViewById<TextView>(R.id.meetingUrlTxt)
```

Comment:
I made the variables private as it is best practice when using a variable that should only be used inside the class it is currently in. I also explicitly declared the variables to prevent null-ability issues.
Source: https://stackoverflow.com/questions/48820066/why-do-we-declare-private-variables-in-java